smoke_test.js

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env node
```

Comment:
This whole file is not we what we are classifying as a smoke test check here: https://github.com/streamr-dev/streamr-infra/wiki/Docker-Smoke-Test#smoke-test